src/routes/login.js

```diff
@@ -1,5 +1,5 @@
 const router = require('express').Router();
-const User = require("../models/user.model");
+const Student = require("../models/Student.model");
```

Comment:
constant usually starts with lower letters not upper ones. No consistency in the files.